setup.py

```diff
@@ -7,7 +7,8 @@
 install_requires = ['neo>0.3.3',
                     'numpy>=1.8.2',
                     'quantities>=0.10.1',
-                    'scipy>=0.14.0']
+                    'scipy>=0.14.0',
+                    'six']
```

Comment:
No real reason. `Six` is not updated since 2015. Anyway pip would grab version 1.10.0 which is the latest one.